lib/Service/ActionScheduler.php

```diff
@@ -27,9 +28,10 @@ public function __construct(
 	}
 
 	/**
-	 * @param ActionType::* $type
+	 * @param string $type
```

Comment:
was it raising issues in the previous state?

Reply:
IDE was complaining :shrug: I can revert it if you want